config/default.yml

```diff
@@ -46,82 +45,82 @@ RSpec/ExampleWording:
   IgnoredWords: []
 
 RSpec/EmptyExampleGroup:
-  Description: 'Checks for `describe` and `context` groups without tests.'
+  Description: Checks if an example group does not include any tests
   Enabled: true
   CustomIncludeMethods: []
 
 RSpec/ExpectActual:
-  Description: 'Checks for `expect(...)` calls containing literal values'
+  Description: Checks for `expect(...)` calls containing literal values
   Enabled: true
 
 RSpec/MultipleDescribes:
-  Description: 'Checks for multiple top level describes.'
+  Description: Checks for multiple top level describes
   Enabled: true
 
 RSpec/MultipleExpectations:
-  Description: 'Checks for multiple `expect(...)` calls in one example.'
+  Description: Checks if examples contain too many `expect` calls
   Enabled: true
   Max: 1
 
 RSpec/NestedGroups:
-  Description: 'Checks for multiple levels of context nesting.'
+  Description: Checks for nested example groups
   Enabled: true
   MaxNesting: 2
 
 RSpec/InstanceVariable:
-  Description: 'Checks for the usage of instance variables.'
+  Description: Checks for instance variable usage in specs
   AssignmentOnly: false
   Enabled: true
 
 RSpec/LetSetup:
-  Description: 'Checks for `let!` being used for test setup.'
+  Description: Checks for `let!` being used for test setup.
   Enabled: true
 
 RSpec/LeadingSubject:
-  Description: 'Checks for `subject` definitions that come after `let` definitions.'
+  Description: Checks for `subject` definitions that come after `let` definitions
   Enabled: true
 
 RSpec/FilePath:
-  Description: 'Checks the file and folder naming of the spec file.'
+  Description: Checks that spec file paths are consistent with the test subject
   Enabled: true
   CustomTransform:
     RuboCop: rubocop
     RSpec: rspec
 
 RSpec/VerifiedDoubles:
-  Description: 'Prefer using verifying doubles over normal doubles.'
+  Description: Prefer using verifying doubles over normal doubles.
```

Comment:
Sure I was actually going to and got bogged down by other changes in this PR which I ended up abandoning (tl;dr splitting up config like rubocop does into `config/enabled.yml` and `config/default.yml` is impossible with the current monkey patching we do)